catalog/app/containers/Auth/messages.js

```diff
@@ -7,10 +7,6 @@ export default defineMessages({
     id: `${scope}.notificationAuthLost`,
     defaultMessage: 'Authentication lost. Sign in again.',
   },
-  notificationAuthError: {
```

Comment:
even if we don't retry, don't we still want to show an error?

Reply:
the one above will be shown (`'Authentication lost. Sign in again.'`)